server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionValidator.java

```diff
@@ -58,11 +64,6 @@ void validateForUpdate(ConnectionConfig connection, ConnectionConfig saved) {
                         || Objects.equals(connection.getType(), saved.getType()),
                 ErrorCodes.FieldUpdateNotSupported, new Object[] {"connection.type"},
                 "Cannot change field 'connection.type'");
-        PreConditions.validRequestState(
```

Comment:
there is no dialectType filed now, it is includes in connectType

Reply:
so, should we check if the connection's connectType is equals to the saved's connectType

Reply:
> so, should we check if the connection's connectType is equals to the saved's connectType

yes, the connectType exists